_rules/form-control-label-descriptive-cc0f0a.md

```diff
@@ -181,4 +181,5 @@ The element with `aria-labelledby` is not a form field.
 ```
 
 [semantic role]: #semantic-role 'Definition of semantic role'
+[semantic roles]: #semantic-role 'Definition of semantic role'
```

Comment:
I'd change the link reference to `[semantic roles][semantic role] rather than adding a new link definition.